src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -36,6 +37,11 @@
 				:loan-id="loan.id"
 				:show-action-button="true"
 				:use-full-width="true"
+				:atc-tracking-props="{
```

Comment:
@dyersituations  @egoroza I think this would have been better handled by using the add-to-basket event and loan id that is already emitted from this component. We should look into refactoring this specific need to use that emitted event to trigger your tracking call instead of embedding it in the Ui version of KivaClassicLoanCard which now has a new discrepancy across applications.

Reply:
@mcstover Good call, I wasn't aware of that event (I'll need to dig into this existing component more).

@egoroza It would be good to add a new ticket to tackle during the fixathon to refactor this usage to prevent tech debt.

Reply:
thanks!